adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Exceptions/AdalError.cs

```diff
@@ -294,6 +294,11 @@ public static class AdalError
         /// <summary>
         /// Broker redirectUri has incorrect format
         /// </summary>
-        public const string BrokerRedirectUriIncorrectFormat = "broker_redirect_uri_incorrect_format ";
+        public const string BrokerRedirectUriIncorrect = "broker_redirect_uri_incorrect";
```

Comment:
>BrokerRedirectUriIncorrect [](start = 28, length = 26)

You can't change this name... even if the text is off. you can change the content however and make another const with the same name... it will potentially break upgrades